locales/es-ES/messages.lang.php

```diff
@@ -100,14 +100,14 @@
 $GLOBALS['__l10n']["Data imported"]="Datos importados";
 $GLOBALS['__l10n']["Data imported successfully"]="Datos importados exitosamente";
 $GLOBALS['__l10n']["Data Lang."]="Idioma de datos.";
-$GLOBALS['__l10n']["Data language"]="Idioma de datos";
 $GLOBALS['__l10n']["Data Language"]="Idioma de Datos";
+$GLOBALS['__l10n']["Data language"]="Idioma de datos";
 $GLOBALS['__l10n']["date"]="fecha";
 $GLOBALS['__l10n']["Date"]="Fecha";
 $GLOBALS['__l10n']["Default search"]="Búsqueda por defecto";
 $GLOBALS['__l10n']["Default user cannot be deleted"]="El usuario por defecto no puede ser borrado";
 $GLOBALS['__l10n']["Define the default values"]="Define los valores por defecto";
-$GLOBALS['__l10n']["Delete"]="Eliminar";
+$GLOBALS['__l10n']["Delete"]="eliminar";
```

Comment:
I'm guessing this carried over from a previous translation. The capitalization appears incorrect.